### PR TITLE
[bug 1148425] Warn people when their locale is not supported

### PR DESCRIPTION
**Pull request by rehandalal**
We only care about 81277bb onwards.

r?

### CODE REVIEW

File: app/js/launch_checks.js [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
 
-      User.get_temporary_user().then(function(user) {
-        if (user) {
-          Navigation.go_to_view('email_confirmation.html');
+        if (!launched && !supported_locale) {
+          Navigation.go_to_view('unsupported_locale.html');
+          return;
         }
+
+        // asyncStorage.setItem('has_launched', true);
```
Comment (mythmon) [inline app/js/launch_checks.js]: ?

Comment (rehandalal) [inline app/js/launch_checks.js]: ah! oops! forgot to uncomment.

File: app/js/user.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 /* global _, SumoDB, asyncStorage */
+/* global _, SumoDB, asyncStorage */
```
Comment (mythmon) [inline app/js/user.js]: It looks like you doubled this line?

File: app/unsupported_locale.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  <!-- localization -->
+  <link rel="prefetch" type="application/l10n" href="data/locales.ini" />
+  <link rel="prefetch" type="application/l10n" href="data/date/date.ini" />
+  <script src="js/libs/l10n.js" defer></script>
+  <script src="js/libs/l10n_date.js" defer></script>
+</head>
+<body class="skin-organic">
+<section role="region">
+  <header>
+    <h1>Unsupported Language</h1>
```
Comment (mythmon) [inline app/unsupported_locale.html]: Putting  these strings in html is going to make localization really hard (read: I have no idea how to do it). Can we put them in a nunjucks or javascript file?

Comment (mythmon) [inline app/unsupported_locale.html]: Also, once they are there, wrapping them in gettext calls would be nice.

Comment (mythmon) [inline app/unsupported_locale.html]: I retract my comments. We'll just have to add html localization.

**Comment by rehandalal** [comment]
updated ^^

**Comment by mythmon** [comment]
That works. r+

**Pull request merged**